[2022-08-29T22:05:07.629-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2022-08-30T02:04:56.800202+00:00 [queued]>
[2022-08-29T22:05:07.645-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2022-08-30T02:04:56.800202+00:00 [queued]>
[2022-08-29T22:05:07.645-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T22:05:07.645-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-08-29T22:05:07.645-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T22:05:07.696-0400] {taskinstance.py:1389} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2022-08-30 02:04:56.800202+00:00
[2022-08-29T22:05:07.701-0400] {standard_task_runner.py:52} INFO - Started process 16847 to run task
[2022-08-29T22:05:07.708-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'create_postgres_table', 'manual__2022-08-30T02:04:56.800202+00:00', '--job-id', '53', '--raw', '--subdir', '/home/augenz/PycharmProjects/airflow/dags/dag_with_postgres_operator_v02.py', '--cfg-path', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpvcvikyfa', '--error-file', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpk3kb0_62']
[2022-08-29T22:05:07.711-0400] {standard_task_runner.py:80} INFO - Job 53: Subtask create_postgres_table
[2022-08-29T22:05:07.891-0400] {task_command.py:371} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2022-08-30T02:04:56.800202+00:00 [running]> on host pop-os.localdomain
[2022-08-29T22:05:08.110-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v01
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2022-08-30T02:04:56.800202+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-30T02:04:56.800202+00:00
[2022-08-29T22:05:08.145-0400] {base.py:68} INFO - Using connection ID 'database-server_postgres' for task execution.
[2022-08-29T22:05:08.163-0400] {sql.py:315} INFO - Running statement: CREATE TABLE IF NOT EXISTS dag_runs (
            dt date,
            dag_id character varying,
            primary key (dt, dag_id)
            ), parameters: None
[2022-08-29T22:05:08.166-0400] {postgres.py:94} INFO - NOTICE:  relation "dag_runs" already exists, skipping

[2022-08-29T22:05:08.205-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=create_postgres_table, execution_date=20220830T020456, start_date=20220830T020507, end_date=20220830T020508
[2022-08-29T22:05:08.282-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29T22:05:08.412-0400] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-29T22:05:08.493-0400] {dagrun.py:567} INFO - Marking run <DagRun dag_with_postgres_operator_v01 @ 2022-08-30 02:04:56.800202+00:00: manual__2022-08-30T02:04:56.800202+00:00, state:running, queued_at: 2022-08-30 02:04:56.861779+00:00. externally triggered: True> successful
[2022-08-29T22:05:08.496-0400] {dagrun.py:612} INFO - DagRun Finished: dag_id=dag_with_postgres_operator_v01, execution_date=2022-08-30 02:04:56.800202+00:00, run_id=manual__2022-08-30T02:04:56.800202+00:00, run_start_date=2022-08-30 02:04:57.852167+00:00, run_end_date=2022-08-30 02:05:08.495999+00:00, run_duration=10.643832, state=success, external_trigger=True, run_type=manual, data_interval_start=2022-08-29 00:00:00+00:00, data_interval_end=2022-08-30 00:00:00+00:00, dag_hash=220c29d1338056f01514245527a7c9a4
